presto-hive-metastore/src/main/java/com/facebook/presto/hive/MetastoreClientConfig.java

```diff
@@ -194,4 +197,30 @@ public MetastoreClientConfig setRequireHadoopNative(boolean requireHadoopNative)
         this.requireHadoopNative = requireHadoopNative;
         return this;
     }
+
+    public boolean isMetastoreImpersonationEnabled()
+    {
+        return metastoreImpersonationEnabled;
+    }
+
+    @Config("hive.metastore.impersonation.enabled")
+    @ConfigDescription("Should Presto user be impersonated when communicating with Hive Metastore")
+    public MetastoreClientConfig setMetastoreImpersonationEnabled(boolean metastoreImpersonationEnabled)
+    {
+        this.metastoreImpersonationEnabled = metastoreImpersonationEnabled;
+        return this;
+    }
+
+    public String getMetastoreDefaultImpersonationUser()
+    {
+        return metastoreDefaultImpersonationUser;
+    }
+
+    @Config("hive.metastore.impersonation.user")
```

Comment:
hive.metastore-impersonation-user

Reply:
same, we have similar naming convention here. "hive.hdfs.impersonation.enabled"